### ISSUE TITLE
Adding a Dedicated Folder To Each User 

### ISSUE THREAD

**Issue by Itailevi420** [wonderfloyd/Hackathon-Corona#39]
add the ability to  upload images to users database folder

**Comment by reuvenz1993** [comment]
issue Title and desc are not the same.
isn't the function to upload images to user dedicated folder is: #38  ?

**Comment by Itailevi420** [comment]
> issue Title and desc are not the same.
> isn't the function to upload images to user dedicated folder is: #38 ?

Yes, but here #39  is more about the UI/UX, so issue #38 discuss adding the functionality and here #39 going to answer the question were to put the image ( maybe a sidebar, slide bar, a whole dedicated page, on the bottom of the page etc.)

**Comment by Itailevi420** [comment]
And yes the description is not so informative. ( It is a little misleading 🙃)

**Comment by wonderfloyd** [comment]
I didn't mean to close the issue

**Comment by Itailevi420** [comment]
Hi @reuvenz1993 ,
First off great job.
I just git pulled the repo and added the new photos table,
when I npm start myPhoto_api I get a knex error 
![Screenshot_2020-05-05_17-38-24](https://user-images.githubusercontent.com/50306464/81078735-455cd700-8ef7-11ea-8176-cab5c43ba909.png)
do you have any idea why or where to start debugging?

**Comment by Itailevi420** [comment]
> Hi @reuvenz1993 ,
> First off great job.
> I just git pulled the repo and added the new photos table,
> when I npm start myPhoto_api I get a knex error
> ![Screenshot_2020-05-05_17-38-24](https://user-images.githubusercontent.com/50306464/81078735-455cd700-8ef7-11ea-8176-cab5c43ba909.png)
> do you have any idea why or where to start debugging?

The problem was a bug in 2 dependencies. I upgraded them to newer versions, 
and it works Haraaa !!!  ; }
